Xamarin.Forms.Material.iOS/MaterialFrameRenderer.cs

```diff
@@ -137,6 +138,7 @@ protected virtual void ApplyTheme()
 
 			// this is set in the theme, so we must always disable it
 			Interactable = false;
+			UpdateCornerRadius();
```

Comment:
Could you remove `UpdateCornerRadius()` from `SetElement` method so as not to call this twice.